### ISSUE TITLE
Sd card format

### ISSUE THREAD

**Issue by samuelhermann** [Z4urce/flipperzero-camera#3]
First of all thanks for the work!
When pressing glipper zero Ok button I heard the bip but no pictures are saved into the esp32cam sd slot.

Which memory card you are using and how is it formated?

Also nice feature to have will be a portrait mode for the camera

Thanks again.

**Comment by ricsim** [comment]
Same here ... green LED, sound, but no files in DCIM

**Comment by eried** [comment]
pictures are saved to the flipper sd

**Comment by ricsim** [comment]
> pictures are saved to the flipper sd

I know that shey SHOULD save there ... but for me they are NOT.
It created a DCIM directory on the Flipper SD card, when I press the OK, it will light up green, and beep, but no BMP picture is created.

**Comment by eried** [comment]
yes but @samuelhermann says "esp32cam sd slot", I am saying they are saved to the flipper sd instead

**Comment by ricsim** [comment]
Yepp ... sorry did not read it carefully.
But for me the BMP is also nicht created on the SD in the Flipper.
It creates the DCIM directory, but that dir is always empty.

**Comment by eried** [comment]
format the card, the fap will create the folder. Maybe try another card? it might be corrupted
